lib/components/narrative/metro/metro-error-renderer.tsx

```diff
@@ -83,11 +83,11 @@ const ErrorRenderer = ({
           return (
             <Container key={error}>
               <Icon Icon={ExclamationCircle} size="3x" />
-              <h3>
+              <h2>
```

Comment:
Why did we bump this up? I think h3 is cleaner

Reply:
a11y issues because the headers were not sequential. I think we never displayed an error in the a11y tests before and now we do?